src/main/java/com/liferay/faces/util/context/FacesContextHelperUtil.java

```diff
@@ -1197,7 +1291,10 @@ public static void setSessionAttribute(FacesContext facesContext, String name, O
 
 	/**
 	 * Gets the underlying/wrapped FacesContext ThreadLocal singleton instance.
+	 *
+	 * @deprecated  Call {@link FacesContext.getCurrentInstance()} instead.
 	 */
+	@Deprecated
```

Comment:
The FacesContextHelperImpl.getFacesContext() method should be `@Deprecated` too I think.

Reply:
@stiemannkj1 And also FacesContextHelper.getFacesContext() as well?

Reply:
I deprecated because the method was incorrectly not `static`. The method isn't really necessary for `FacesContextHelperUtil` because `FacesContext.getCurrentInstance()` exists. However, it may be useful to have `FacesContextHelper.getFacesContext()`; for example if we were to change `FacesContextHelper` to save off the `FacesContext`, then `getFacesContext()` would make sense and be better/faster than `FacesContext.getCurrentInstance()`.

Reply:
So I think we should keep `FacesContextHelper.getFacesContext()` non-deprecated for now.